source/opt/loop_unroller.cpp

```diff
@@ -842,19 +851,21 @@ void LoopUnrollerUtilsImpl::AssignNewResultIds(BasicBlock* basic_block) {
   }
 }
 
-// For all instructions in |basic_block| check if the operands used are from a
-// copied instruction and if so swap out the operand for the copy of it.
-void LoopUnrollerUtilsImpl::RemapOperands(BasicBlock* basic_block) {
-  for (Instruction& inst : *basic_block) {
-    auto remap_operands_to_new_ids = [this](uint32_t* id) {
-      auto itr = state_.new_inst.find(*id);
+void LoopUnrollerUtilsImpl::RemapOperands(Instruction* inst) {
+  auto remap_operands_to_new_ids = [this](uint32_t* id) {
+    auto itr = state_.new_inst.find(*id);
 
-      if (itr != state_.new_inst.end()) {
-        *id = itr->second;
-      }
-    };
+    if (itr != state_.new_inst.end()) {
+      *id = itr->second;
+    }
+  };
+
+  inst->ForEachInId(remap_operands_to_new_ids);
+}
 
-    inst.ForEachInId(remap_operands_to_new_ids);
+void LoopUnrollerUtilsImpl::RemapOperands(BasicBlock* basic_block) {
+  for (Instruction& inst : *basic_block) {
```

Comment:
Or just move the original comment here.  It seems to still apply.

Reply:
Yah, generally the method comment describes behaviour from the perspective of the caller.  Comments about the algorithm's internal operation (that are not relevant to the interface) belong inside the function body.